docs/internet-identity-spec.adoc

```diff
@@ -520,7 +523,8 @@ This flow is the boring default
 3. User sees their management screen. In particular
 +
 - Their user number
-- The list of their devices, with device aliases, and a button to remove
+- A button to add additional devices
```

Comment:
Or two buttons, one to add login devices and one to add recovery devices?

Reply:
It's a little more.. complicated by now:
![grafik](https://user-images.githubusercontent.com/6189397/119134570-988bc400-ba3d-11eb-977f-9a60fa10a75d.png)

The nag is only shown when no recovery has been configured so far, otherwise you get:
![grafik](https://user-images.githubusercontent.com/6189397/119134642-b3f6cf00-ba3d-11eb-87cd-e34a72312343.png)

Reply:
Great, well done!